src/components/LoginOptions/LoginOptions.stories.mdx

```diff
@@ -0,0 +1,178 @@
+import { Canvas, Meta, Story, Props, ArgsTable } from '@storybook/addon-docs';
+
+import {LiteralsProvider} from 'components/Literal';
+
+import LoginOptions from '.';
+import LoginOptionsDisplay from './LoginOptionsDisplay';
+
+<Meta
+  title="Composites/Login Options"
+  component={LoginOptions}
+/>
+
+# Login Options
+
+The `LoginOptions` component displays the options that are available for a user to authenticate when starting a form.
+
+## Presentation
+
+The `LoginOptionsDisplay` component shows the different buttons that can be clicked to start the authentication process.
+Next to each button, the corresponding logo is displayed.
+
+export const DisplayTemplate = ({loginAsYourselfOptions, loginAsGemachtigdeOptions}) => (
+  <LoginOptionsDisplay
+    loginAsYourselfOptions={loginAsYourselfOptions}
+    loginAsGemachtigdeOptions={loginAsGemachtigdeOptions}
+  />
+);
+
+<Canvas>
+  <Story
+    name="Display"
+    args={{
+      loginAsYourselfOptions: [
+        {
+          identifier: "digid",
+          label: "DigiD",
+          logo: {
+              href: "https://www.digid.nl/",
+              title: "DigiD",
+              imageSrc: "/digid.png"
+          },
+          url: "http://localhost:8999/auth/form-slug/digid/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+        },
+        {
+          identifier: "eHerkenning",
+          label: "eHerkenning",
+          logo: {
+              href: "https://www.eherkenning.nl/",
+              title: "eHerkenning",
+              imageSrc: "/eherkenning.png"
+          },
+          url: "http://localhost:8999/auth/form-slug/eherkenning/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+        },
+        {
+          identifier: "eidas",
+          label: "eIDAS",
+          logo: {
+              href: "https://www.eherkenning.nl/",
+              title: "eIDAS",
+              imageSrc: "/eidas.png"
+          },
+          url: "http://localhost:8999/auth/form-slug/eidas/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+        },
+        {
+          identifier: "anonymous",
+          label: "Begin form",
+          url: "#",
+        }
+      ],
+      loginAsGemachtigdeOptions: [{
+	      identifier: "digid_machtigen",
+	      label: "DigiD Machtigen",
+        logo: {
+            href: "https://www.digid.nl/",
+            title: "DigiD Machtigen",
+            imageSrc: "/digid.png"
+        },
+        url: "http://localhost:8999/auth/issue-2447-style-login-page/digid_machtigen/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+      }]
+    }}
+  >
+    {DisplayTemplate.bind({})}
```

Comment:
perhaps adding a `<form>` container around this (possibly as a decorator?) makes sense to simulate realistic behaviour:

```suggestion
    <form onSubmit={e => {
      e.preventDefault();
      alert('form start triggered');
    }}>
      {DisplayTemplate.bind({})}
    </form>
```

This also better documents the expected usage (inside of a `<form>` container)